contents/blog/helping-engineers-to-product.md

```diff
@@ -0,0 +1,68 @@
+
+### Be the pipeline for product and customer data
+
+Traditionally, as companies scale their data warehouse becomes the source of truth and non-warehouse native tools (like product analytics) become less relevant. By being their core pipeline from connecting their data to their warehouses we can remain sticky for the life of our customers. And by providing this infra, we ensure the data we have remains comprehensive. We will continue pushing back the need for companies to even set up a warehouse in the first place.
+
+## So what's to come this year?
+
+Right now, we're working on a slicker core experience –  we've got many team members working on reliability, scalability, and data load times. We're adding power features for more technical users, like [HogQL and SQL access](https://github.com/PostHog/meta/issues/81) inside the product. We're also plugging gaps in our product, such as [iOS session recording](https://github.com/PostHog/posthog/issues/12344), [json feature flags](https://github.com/PostHog/posthog/pull/13623), [feature flag resilience](https://github.com/PostHog/posthog/issues/13601) and improved SDK coverage for feature flags. Finally, we're pushing for [PostHog 3000](https://github.com/PostHog/posthog/issues/12923) – a big UI change that will ultimately connect all our tools together better.
+
+Next quarter, we'll be working on our CDP – we want to make that a first class citizen product – like our analytics or session recording. Today, we have 50+ "apps", largely community-driven, that mainly integrate PostHog with other data sources, but we want to add more integrations, deepen their functionality, improve their reliability, and make them more intuitive to use.
+
+It's not all product: 
+
+- In our growth team, we made a lot of progress last year on conversion to revenue, which grew 6x, but we're now looking at ways to give away as much as we can, for free, to _get in first_. 
+
+- In our customer success team, we're getting more targeted in our approach so we focus on high-growth companies.
+
+- In marketing, we aim to produce the best content on the internet for each piece that we write. We're moving paid ad spend to hire another writer, we're producing more tutorials than ever before, and we're focusing more of our writing more tightly on how to help engineers learn skills outside of coding.
+
+## I'm excited
+
+Ali, on our board, told us "when you get bigger, you can see around corners". That turns out to be true. Now we've tried a bunch of stuff, we have 23,000+ companies who have installed PostHog and approaching 70,000 developers in the community, we can clearly see what we need to do. And, seeing the results we've had so far, increases our confidence we can make it all happen.
```

Comment:
```suggestion
Ali, on our board, told us "when you get bigger, you can see around corners". That turns out to be true. Now we've tried a bunch of stuff achieved product-market fit – we have 23,000+ companies who have installed PostHog and approaching 70,000 developers in the community – we can clearly see what we need to do. And, seeing the results we've had so far, increases our confidence we can make it all happen.
```

Reply:
don't think your edi tmakes sense, want to clarify?

Reply:
Oh, there's a missing 'and' – i.e:

> Now we've tried a bunch of stuff and achieved product-market fit – we have 23,000+ companies who have installed PostHog and approaching 70,000 developers in the community – we can clearly see what we need to do